### PR TITLE
Fix browser tests that fail on Chrome #7699

### LINKED ISSUE #7699: Fix browser tests that fail on Chrome
Even though we [support Chrome](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#using-chrome), 3 browser tests are failing, namely:
```
AdminEmailPageUiTest
InstructorFeedbackEditPageUiTest
InstructorFeedbackResultsPageUiTest
```

### PR DESCRIPTION
**Pull request by whipermr5**
Fixes #7699

**Outline of Solution**
- Fix browser tests on Chrome
- Use Chrome for AppVeyor build
  - Remove Firefox install from `appveyor.yml`
  - Add Travis- and AppVeyor-specific test properties files

**Results**
- AppVeyor build (Chrome) is [passing](https://ci.appveyor.com/project/whipermr5/teammates/build/1.0.221)
  - Using Chrome decreases the build time by 3 minutes 🎉
    - Edit: this is probably due to some tests being [skipped](https://github.com/TEAMMATES/teammates/blob/99aaeabcbbf95cdbcbb40b1c80377d9ec87530c7/src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java#L353-L356)
- Travis build (Firefox) is [passing](https://travis-ci.org/whipermr5/teammates/builds/252042555)

### CODE REVIEW

**Comment by LiHaoTan** [comment]
>Using Chrome decreases the build time by 3 minutes

Holy really? Should we switch Travis to use Chrome instead and Appveyor to use Firefox then :P?

**Comment by whipermr5** [comment]
@LiHaoTan I realised some tests are skipped when Chrome is detected, so there may not be any actual speedup 😞

**Comment by whipermr5** [comment]
@wkurniawan07 Reminder :)

File: src/test/java/teammates/test/driver/TestProperties.java [hunk 1]
```diff
@@ -86,7 +86,15 @@
...
-            prop.load(new FileInputStream("src/test/resources/test.properties"));
+            String propertiesFilename;
+            if (isTravis()) {
+                propertiesFilename = "test.travis.properties";
+            } else if (isAppveyor()) {
+                propertiesFilename = "test.appveyor.properties";
+            } else {
+                propertiesFilename = "test.properties";
+            }
+            prop.load(new FileInputStream("src/test/resources/" + propertiesFilename));
```
Comment (wkurniawan07) [inline src/test/java/teammates/test/driver/TestProperties.java]: Again, I don't feel like we need to go this far (having separate `.properties` files) just for this. Is it possible to just do everything via `TestProperties.java`?
```java
...
// set all properties as before
if (isTravis()) {
    ...
}
if (isAppveyor()) {
    ...
}
...
```

Comment (whipermr5) [inline src/test/java/teammates/test/driver/TestProperties.java]: I don't like the idea of putting build-specific configuration values inside Java code. We should allow easy tweaking via build config files without having to modify code.

Comment (whipermr5) [inline src/test/java/teammates/test/driver/TestProperties.java]: E.g. are we really going to hardcode the path to the chromedriver binary in the AppVeyor environment inside our codebase? We'd have to change the code whenever their environment changes.

Comment (wkurniawan07) [inline src/test/java/teammates/test/driver/TestProperties.java]: The only differing property is the browser. I don't think it's too much to modify the code to selectively choose the browser based on the environment. Everything else can be hardcoded as the default, just like how it was done before.

Comment (whipermr5) [inline src/test/java/teammates/test/driver/TestProperties.java]: If we hardcode the AppVeyor chromedriver binary path into the template, it feels misleading. It's also not very explicit that it's the AppVeyor path, as it's found in the general template.

I quite strongly feel that a template file should be used as a reference and not be depended upon to provide essential values in a build environment. What if we decide to switch Travis to Chrome too in future? We'd need two paths to the chromedriver binary that are specific to their build environments.

Besides, may I ask what inhibitions you have about creating separate properties files per build environment? I find this very clear and readable as we'd know exactly what test settings we have for each environment, without delving into the code of `TestProperties`.

Comment (whipermr5) [inline src/test/java/teammates/test/driver/TestProperties.java]: Reminder :)

**Pull request merged**